types/index.d.ts

```diff
@@ -14,7 +14,6 @@
 
 // FIXME: Is this `any` really necessary?
 type VariableArgFunction = (...params: any[]) => unknown;
-type ArgumentTypes<F extends VariableArgFunction> = F extends (...args: infer A) => unknown ? A : never;
```

Comment:
Is it necessary to remove these from `index.d.ts`? I know we don't implement them right now, but I would like to keep them around because we _are_ planning to implement all of them for compatibility.

Reply:
Well, I removed the type as it is the same as the built-in `Parameters` type. Do you want me to re-add it?